__tests__/__system__/__scripts__/cm_create_v2.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+zowe config init --prompt=false > /dev/null 2>&1
+
+zowe config list "profiles" --rfj
```

Comment:
Before listing the contents of team config I think we should add a user and password to the profile so we can verify they are stored securely, similar to the test for v1 profiles.

We could do that with the following commands:
```
zowe config set profiles.base.properties.user USER
zowe config set profiles.base.properties.password PLAINTEXT
```

Then in the unit test, we could `expect` that the plain text values "USER" and "PLAINTEXT" are not contained in stdout, but the placeholder for secure values "(secure value)" is.